### PR TITLE
added react-router-dom

### PR DESCRIPTION
**Pull request by RSmetanyuk**

### CODE REVIEW

**Comment by RSmetanyuk** [comment]
[Add React Router to the project](https://trello.com/c/Tm3kx8bo/5-add-react-router-to-the-project)

File: src/App.js [hunk 1]
```diff
@@ -1,18 +1,51 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
 import logo from './logo.svg';
 import './App.css';
 
-function App() {
+const Home = () => (
```
Comment (i10111pn) [inline src/App.js]: Please, add this to a separate file and use the ES6 class.

File: src/App.js [hunk 2]
```diff
@@ -1,18 +1,51 @@
...
+};
+
+function App() {
+  return (
+    <Router>
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <p>
+            React/Nuxeo/Docker app (2).
```
Comment (i10111pn) [inline src/App.js]: Can you update '(2)' with '(v 0.3)'

File: src/App.js (hunk 1 above)
Comment (RSmetanyuk) [reply src/App.js]: Done

File: src/App.js (hunk 2 above)
Comment (RSmetanyuk) [reply src/App.js]: Done

**Comment by RSmetanyuk** [comment]
Should I merge pull request?

**Comment by i10111pn** [comment]
Sure.

**Pull request merged**